src/backend/internal/middlewares/jwt.go

```diff
@@ -59,10 +59,32 @@ func JWTMiddleware(DB *gorm.DB,secret string, secureCookie bool, httpOnly bool)
 			Unauthorized: func(c *gin.Context, code int, message string) {
 				log.Error(errors.NewBadRequest(message))
 				c.JSON(code, gin.H{
-					"code":    code,
+					"status":    code,
 					"message": message,
 				})
 			},
+			LoginResponse: func(c *gin.Context, code int, token string, expire time.Time) {
+				c.JSON(http.StatusOK, gin.H{
+					"status":   http.StatusOK,
+					"message": "Logged in successfully",
+					"token":  token,
```

Comment:
@Robogeek95  nice one here,

But the format should be 

```
{
"status":   http.StatusOK,
"message": "Logged in successfully",
"data": {
    "token":  token,
    "expire": expire.Format(time.RFC3339),
    },
"error":null

}
```

Reply:
Yea, I thought about that too but I want us to deliberate on in. I don't want to impose the format

Reply:
okay cool, that is the format used by most projects